### PR TITLE
Add custom code tab to timeline screen.

### PR DESCRIPTION
**Pull request by albertgcsula**
Add custom code tab to timeline screen, if schema has the mixin.

### CODE REVIEW

File: src/plugins/curator/screens/timeline/index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ class TimelineScreen extends AbstractNodeScreen {
   getTabs() {
     return [
       'details',
+      delegate.getSchemas().node.hasMixin('triniti:common:mixin:custom-code') && 'code',
```
Comment (gdbrown) [inline src/plugins/curator/screens/timeline/index.js:17]: @albertgcsula use schemas that is already available (like the seo check below)

Comment (albertgcsula) [reply src/plugins/curator/screens/timeline/index.js]: @gdbrown updated.

**Comment by efjacobson** [comment]
wait this needs something else

**Comment by albertgcsula** [comment]
> @albertgcsula you need to add the `code` bit to the curator `routes.js`

Thanks for catching that @efjacobson The code is updated.

**Pull request merged**